django/forms/boundfield.py

```diff
@@ -177,7 +177,6 @@ def label_tag(self, contents=None, attrs=None, label_suffix=None):
                 else:
                     attrs['class'] = self.form.required_css_class
         context = {
-            'form': self.form,
```

Comment:
It's enough to just remove `form` from here. 

I think folk will find having the `field` available useful. For example, I envisage folk will want to do something like this:

``` django
<label class="{% if field.field.required %} requiredField{% endif %}">
  {{ label }}
</label>
 ```